specification/sections/05-events.md

```diff
@@ -229,9 +229,11 @@ The SDK must update it's internal representation of the provider's state accordi
 | -------------------------------- | ------------------------------------------------------- |
 | `PROVIDER_READY`                 | `READY`                                                 |
 | `PROVIDER_STALE`                 | `STALE`                                                 |
-| `PROVIDER_ERROR`                 | `ERROR`                                                 |
+| `PROVIDER_ERROR`                 | `ERROR`/`FATAL`*                                        |
 | `PROVIDER_CONFIGURATION_CHANGED` | N/A (provider remains in its current state)             |
 | `PROVIDER_CONTEXT_CHANGED`       | N/A (only emitted by SDK during context reconciliation) |
 | `PROVIDER_RECONCILING`           | N/A (only emitted by SDK during context reconciliation) |
 
-see: [provider lifecycle management](01-flag-evaluation.md#17-provider-lifecycle-management)
\ No newline at end of file
+\* If the `error code` associated with the error indicates `PROVIDER_FATAL`, the state is set to `FATAL`
```

Comment:
This is mentioned already in [1.7.5](https://github.com/open-feature/spec/blob/ae286cf5192b4e341ea691caee6a4c95582f32ed/specification.json#L327-L331), but worth making clear in this table.